agent/Makefile

```diff
@@ -260,7 +260,7 @@ install-lib:
 install-python3-setup: install-util-scripts install-lib
 	${COPY} requirements.txt ${DESTDIR}
 	mkdir -p ${DESTDIR}/python3
-	(cd ..; SKIP_GENERATE_AUTHORS=1 SKIP_WRITE_GIT_CHANGELOG=1 python3 setup.py install --prefix=${DESTDIR}/python3)
+	(cd ..; /usr/bin/python3 -m pip install --prefix=${DESTDIR}/python3 -e .)
```

Comment:
Just as an aside, we don't need the parens, here -- this is a `make` recipe and not a script, so each line is run in a subshell already -- and `&&` would be better than `;` between the commands (not that the `cd` command is likely to ever fail, but as a general practice).

Reply:
Sure, but I believe this is a straight back-port of the commit from `main`.  It looks like another commit addressed your concerns (see PR #2961), but I didn't feel it was worth trying to back-port change pulling it out of the other changes around it.